docs/tutorials/clustering/kclust.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.9-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# K-Means Clustering"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import random\n",
+    "random.seed(10)"
+   ]
+  },
+  {
+   "source": [
+    "K-Means Clustering (kclust) is a clustering algorithm that finds the optimal kmeans clustering model by iterating over a range of values and creating a model with the lowest possible silhouette score.\n",
+    "\n",
+    "Kmeans clustering is a form of clustering that works to seperate samples into a number of equivariant groups while minimizing the sum of the smallest squared deviations that each sample point has from any of the cluster means. Mathematically, the algorithm is trying to create clusters with small enough means to minimize the following quantity:\n",
+    "\n",
+    "$\\sum_{i=1}^{N}$ min$_{\\mu_{c}^{c \\in C}}||x_{i} - \\mu_{c}||^{2}$\n",
```

Comment:
This equation should be centered using double dollar signs. Also, for operators like min, you can use underset as follows to get the text actually under the operator: ``\underset{\mu_c, c \in C}{\text{min}} ||x_i - \mu_c||^2``.